controllers/handlers_utils.go

```diff
@@ -110,9 +110,10 @@ func deployContentOfConfigMap(ctx context.Context, deployingToMgmtCluster bool,
 	destClient client.Client, configMap *corev1.ConfigMap, clusterSummary *configv1beta1.ClusterSummary,
 	mgmtResources map[string]*unstructured.Unstructured, logger logr.Logger,
 ) ([]configv1beta1.ResourceReport, error) {
-
+	subressources := strings.Split(configMap.GetObjectMeta().GetAnnotations()["projectsveltos.io/subressources"], ",")
```

Comment:
we can move this to a method as we also need to check that annotations is not nil